newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -229,86 +210,38 @@ export default function SummaryPage() {
-            </Modal.Header>
-            <Modal.Content>
-              <Trans>
-                Do you really wish to delete this newdle? Please be aware that{' '}
-                <strong>this action cannot be undone.</strong>
-              </Trans>
-              {hasParticipantsWithEmail && (
-                <div className={styles['mail-checkbox']}>
-                  <Checkbox
-                    checked={sendDeletionMail}
-                    onChange={() => setSendDeletionMail(!sendDeletionMail)}
-                    label="Notify the participants via e-mail"
-                  />
-                  {sendDeletionMail && (
-                    <Input
-                      type="text"
-                      placeholder={t`Leave a comment (optional)`}
-                      className={styles['deletion-comment']}
-                      value={comment}
-                      onChange={(__, {value}) => setComment(value)}
-                    />
-                  )}
-                  {hasParticipantsWithoutEmail && (
-                    <Message className={styles['email-participant-list']}>
-                      <strong>
-                        <Trans>
-                          Some of your participants do not have e-mail addresses and will not be
-                          contacted:
-                        </Trans>
-                      </strong>
-                      <RecipientList
-                        recipients={participantsWithoutEmail}
-                        color="red"
-                        icon="close"
-                      />
-                    </Message>
-                  )}
-                </div>
-              )}
-            </Modal.Content>
-            <Modal.Actions>
+            <div className={styles['button-row']}>
               <Button
-                negative
-                onClick={() => handleDeletionModalConfirm(true)}
-                loading={deletionMailSending || deleting}
+                type="submit"
+                loading={submitting}
+                disabled={!finalDate}
+                className={styles['finalize-button']}
+                onClick={update}
               >
-                <Trans>Delete newdle</Trans>
+                <Trans>Select final date</Trans>
               </Button>
-            </Modal.Actions>
-          </Modal>
+              <Dropdown
+                text={t`Edit`}
+                floating
+                button
+                direction="right"
+                color="violet"
+                as={Button}
+                className={styles['edit-button']}
+              >
+                <Dropdown.Menu>
+                  <Dropdown.Item as={Link} text={t`Participants`} to={`${editUrl}/participants`} />
+                  <Dropdown.Item as={Link} text={t`Timeslots`} to={editUrl} />
+                  <Dropdown.Item as={Link} text={t`Options`} to={`${editUrl}/options`} />
+                  <Dropdown.Item
```

Comment:
The dropdown is not visible for finalized newdles (since editing those makes no sense), so we need to show a standalone delete button in that case.